### PR TITLE
#295 - Adding MaxLinear Evaluation Board

### PR DESCRIPTION
**Pull request by punnypenguins**
Issue #295

### CODE REVIEW

File: _boards_cm/maxlinear-evaluation-board.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: board
```
Comment (geerlingguy) [inline _boards_cm/maxlinear-evaluation-board.md:2]: This one is a `card`, and right now functionality is 'maybe', and driver required is also 'maybe' :)

**Review by geerlingguy — CHANGES_REQUESTED** [review]

File: _boards_cm/maxlinear-evaluation-board.md (hunk 1 above)
Comment (punnypenguins) [reply _boards_cm/maxlinear-evaluation-board.md:2]: Which card folder does it go in, @geerlingguy? I'll get it fixed now. Thanks for bumping it!

Comment (geerlingguy) [reply _boards_cm/maxlinear-evaluation-board.md:2]: It goes under https://pipci.jeffgeerling.com/#other-cards (other).

Comment (punnypenguins) [reply _boards_cm/maxlinear-evaluation-board.md]: @geerlingguy Fixed it! New commit is below.

**Pull request merged**